tasks/concurrent.js

```diff
@@ -4,16 +4,20 @@ var padStream = require('pad-stream');
 var async = require('async');
 var arrify = require('arrify');
 var indentString = require('indent-string');
+var chalk = require('chalk');
 
 var cpCache = [];
 
+var colors = ["blue","magenta","yellow","green","red","cyan","white","gray","redBright","greenBright","yellowBright","blueBright","magentaBright","cyanBright"];
```

Comment:
This needs to be inside the grunt task function since you modify it there, otherwise there will be no color left if this task is used multiple times.